### DISCUSSION TITLE
TSML-US 3.14.1 Cuts Out Top Lines

### DISCUSSION THREAD

**Discussion by Grateful-Viv** [code4recovery/12-step-meeting-list#780] [Q&A] [answered]
My site is setup to have the menu bar set at fixed to top on scroll which shows the menu bar when you are scrolling down a page. 

When first selecting a meeting's detail, it jumps down too far. See picture.

![Fixed to Top](https://user-images.githubusercontent.com/62247421/164746779-62241d92-d34a-4e0d-85ab-90965fcec013.png)

If I change to standard position (not fixed) for menu bar, the page with the meeting detail has the title of the meeting at the top, but it still is blocking the meeting top widget text (that I would love removed on the meeting detail - see #686).

**Comment by joshreisner** [comment, marked as answer]
hi @Grateful-Viv try adding 
```
* {
  scroll-margin-top: 50px;
}
```
you can adjust more or less depending on how much you need.

i know the widget request is underway

**Reply by Grateful-Viv** [reply]
Thanks. That works.
